src/cl/nimbus/nimbus_launcher.star

```diff
@@ -22,7 +22,7 @@ BEACON_MIN_CPU = 50
 BEACON_MIN_MEMORY = 256
 
 DEFAULT_BEACON_IMAGE_ENTRYPOINT = ["nimbus_beacon_node"]
-
+BEACON_NODE_ENTRYPOINT = "/home/user/nimbus-eth2/build/nimbus_beacon_node"
```

Comment:
It looks very wrong hardcoding this path here as its likely gonna change when nimbus touches their build system.
But yea,  don't really see a better solution :/
Do our ethpandaops nimbus images use the same entrypoint?

Reply:
yes, works with our images too. 

We can always change it if they decide to change their entrypoint. For now I think this is our "only" reasonable solution.